QgisModelBaker/libqgsprojectgen/dbconnector/gpkg_connector.py

```diff
@@ -708,13 +717,23 @@ def create_basket(self, dataset_tid, topic):
                     topic
                 )
             try:
+                (
+                    next_id,
+                    fetch_and_increment_feedback,
+                ) = self._fetch_and_increment_key_object(self.tid)
+                if not next_id:
```

Comment:
To be sure: 0 is an invalid id?

Reply:
You are right 0 is used. I've been confused because my datasets got 1 in a freshly created db. But when I add an entry to another table as first action, I start with 0.

Reply:
Considered that